rfcs/proposed/zip_view/README.md

```diff
@@ -0,0 +1,40 @@
+# zip_view Support for the oneDPL Range APIs with C++20
+
+## Introduction
+`std::ranges::zip_view` is a powerful utility that enables developers to combine two or more ranges into a single view,
+where each element is represented as a tuple containing corresponding elements from each input range.
+
+## Motivations
+`std::ranges::zip_view` is a convenient way to combine multiple ranges into a single view, where each element of
+the resulting range is a tuple containing one element from each of the input ranges. This can be particularly
+useful for iterating over multiple collections in parallel. `std::ranges::zip_view` was introduced in C++23,
+but many developers are still using C++20 standard. So, oneDPL introduces `oneapi::dpl::ranges::zip_view`,
+with the same API and functionality as `std::ranges::zip_view`.
```

Comment:
Thinking more of `zip_view`, I now wonder if we should add it to the specification or rather should provide it as a library extension.

If the goal is only to close the gap between C++20 and C++23, while the standard C++23 `zip_view` is well usable with oneDPL, including with device policies - then maybe it is not worth adding our `zip_view` variation to the specification. Perhaps especially so if it cannot be used as a substitution for the standard `zip_view` in all scenarios (that means, if developers cannot always use the oneDPL variation and may need to choose between that and the standard one).